images/oc-build-deploy-dind/build-deploy-docker-compose.sh

```diff
@@ -12,22 +12,26 @@ function outputToYaml() {
 }
 
 function cronScheduleMoreOftenThan15Minutes() {
-  #takes a unexpanded cron schedule, returns 0 if it's more often that 15 minutes
+  # Work out if a particular cron schedule runs at least every 15 minutes.
+  # returns 0 if cron runs at least every 15 minutes. This includes a 15 minute
+  # cron as well. Running a cron every 10 minutes will also return 0.
+  # returns 1 if cron runs less often, say every 30 minutes.
   MINUTE=$(echo $1 | (read -a ARRAY; echo ${ARRAY[0]}) )
   if [[ $MINUTE =~ ^(M|H|\*)\/([0-5]?[0-9])$ ]]; then
-    # Match found for M/xx, H/xx or */xx
-    # Check if xx is smaller than 15, which means this cronjob runs more often than every 15 minutes.
+    # Match found for `M/xx`, `H/xx` or `*/xx`.
+    # Check if xx is smaller or equal to 15, which means this cronjob runs at
+    # least every 15 minutes.
     STEP=${BASH_REMATCH[2]}
-    if [ $STEP -lt 15 ]; then
+    if [ $STEP -le 15 ]; then
       return 0
     else
       return 1
     fi
   elif [[ $MINUTE =~ ^\*$ ]]; then
-    # We are running every minute
+    # We are running every minute.
     return 0
   else
-    # all other cases are more often than 15 minutes
+    # All other cases are less often than 15 minutes.
```

Comment:
I don't think that "less" is correct now, this would mean that "other cases" are 15minute and higher (like 1 hour, 16min, etc.). But we return 1 when we have 14, 13, 12, etc.

So I would leave it at `All other cases are more often than 15 minutes.`, as before it actually would have needed to be called `All other cases are more often than or exactly 15 minutes.`

Reply:
Yeah, the wording in general was doing my head in by the end of this PR. Will change.